docs/Using Fleet/Troubleshooting-live-queries.md

```diff
@@ -112,11 +112,11 @@ If the infrastructure is working correctly but the query is hanging or crashing
 
 To rule this out, you should also try out the dummy query `SELECT 1 WHERE 1 = 0;`.
 If you see "(100% responded)" with the dummy query but not with your query, then this might be an issue with:
-  - the query crashing osquery on some devices (watchdog killing the osquery process).
-  - the query hanging or taking too long to run on some devices.
-  - the query returning too many results (that may reach network limits). Try reducing the number of results by using `LIMIT N;` on the query.
+  - the query is crashing osquery on some devices (e.g., watchdog is killing the osquery process.)
```

Comment:
Since these are no longer completing a sentence started above, we can capitalize them:
```suggestion
  - The query is crashing osquery on some devices (e.g., watchdog is killing the osquery process).
```